portal/src/model/themeAuthFlowV2.ts

```diff
@@ -7,8 +7,42 @@ import {
   CssRuleNodeWrapper,
 } from "../util/cssVisitor";
 
+export enum Theme {
+  Light = "light",
+  Dark = "dark",
+}
+
 export const enum ThemeTargetSelector {
   Light = ":root",
+  Dark = ":root.dark",
+}
+export function getThemeTargetSelector(theme: Theme): ThemeTargetSelector {
+  switch (theme) {
+    case Theme.Light:
+      return ThemeTargetSelector.Light;
+    case Theme.Dark:
+      return ThemeTargetSelector.Dark;
+    default:
+      return ThemeTargetSelector.Light;
+  }
+}
+
+export function selectByTheme<T>(option: { [t in Theme]: T }, theme: Theme): T {
+  return option[theme];
+}
+
+export const enum CSSVariable {
+  LayoutFixAlignItems = "--layout-flex-align-items",
+  LayoutPaddingLeft = "--layout-padding-left",
+  LayoutPaddingRight = "--layout-padding-right",
+  LayoutBackgroundColor = "--layout__bg-color",
+  LayoutBackgroundImage = "--layout__bg-image",
+  PrimaryButtonBackgroundColor = "--primary-btn__bg-color",
```

Comment:
hover and active color also need to be set.
```
--primary-btn__bg-color--hover
--primary-btn__bg-color--active
```

Compute the color by using fluent ui theme generator. We currently use `themeDark` as the active and hover color.

ref: https://developer.microsoft.com/en-us/fluentui?fabricVer=5#/styles/themegenerator